<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ProcessWidget" name="process_widget" native="true"/>
     </item>
     <item>
      <widget class="FinishWidget" name="finish_widget" native="true"/>
     </item>
     <item>
      <widget class="SelectPathWidget" name="select_widget" native="true"/>
     </item>
     <item>
      <widget class="ServerAgreementWidget" name="serveragreemtn_widget" native="true"/>
     </item>
     <item>
      <widget class="WelcomeWidget" name="welcome_widget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WelcomeWidget</class>
   <extends>QWidget</extends>
   <header>welcomewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FinishWidget</class>
   <extends>QWidget</extends>
   <header>finishwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProcessWidget</class>
   <extends>QWidget</extends>
   <header>processwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectPathWidget</class>
   <extends>QWidget</extends>
   <header>selectpathwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ServerAgreementWidget</class>
   <extends>QWidget</extends>
   <header>serveragreementwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
